lib/ui/strings.dart

```diff
@@ -902,4 +902,12 @@ class Strings {
   static String suggestionLocalisationFormImmersionSubtitle =
       "Sélectionnez une ville dans laquelle vous cherchez une immersion.";
   static String suggestionLocalisationAddAlerteButton = "Ajouter l’alerte";
+
+  // a11y
```

Comment:
```suggestion
  // alternatives images et label
```
pour éviter de mettre dans une case a11Y